Sources/SwaggerSwift/parsePath.swift

```diff
@@ -7,44 +7,44 @@ func parse(path: SwaggerSwiftML.Path, servicePath: String, swagger: Swagger, swa
 
     var functions = [NetworkRequestFunction]()
     var modelDefinitions = [ModelDefinition]()
-    if let operation = path.get {
-        let result = parse(operation: operation, httpMethod: .get, servicePath: servicePath, parameters: parameters, swagger: swagger, swaggerFile: swaggerFile, verbose: verbose)
+    if let operation = path.get,
+       let result = parse(operation: operation, httpMethod: .get, servicePath: servicePath, parameters: parameters, swagger: swagger, swaggerFile: swaggerFile, verbose: verbose) {
         functions.append(result.0)
         modelDefinitions.append(contentsOf: result.1)
```

Comment:
```suggestion
       let (parsedFunctions, parsedDefinitions)  = parse(operation: operation, httpMethod: .get, servicePath: servicePath, parameters: parameters, swagger: swagger, swaggerFile: swaggerFile, verbose: verbose) {
        functions.append(parsedFunctions)
        modelDefinitions.append(contentsOf: parsedDefinitions)
```
just for readability

Reply:
totally agree, fixed